### ISSUE TITLE
improvement: more explicit error

### ISSUE THREAD

**Issue by pinage404** [brumhard/masklint#3]
Hi,

i added masklint to another project and i got the error

```txt
masklint run
Error: No such file or directory (os error 2)
```

i forgot to add shellcheck

i would like a more explicit error like "`shellcheck` not found in the `$PATH`"

**Comment by brumhard** [comment]
Ah that's fair, I stumbled over that myself before. Implementation is pretty rough but should be fixed with v0.3.0. Let me know if it works for you.

**Comment by pinage404** [comment]
i updated and got the following message :

```sh
Error: executable for shellcheck not found in $PATH
```

it find it really helpful !

Thanks ! :smile:
